pkg/analysis/builtins.go

```diff
@@ -22,20 +23,45 @@ import (
 type Builtins struct {
 	Functions map[string]query.Signature
 	Symbols   []query.Symbol
+	Types     map[string]query.Class
+	Methods   map[string]query.Signature
+	Members   []query.Symbol
```

Comment:
Can we read these off of `Types` instead? When I saw these fields I couldn't figure out their purpose, and when I got to the code using them, it looked like they were buggy (e.g., when I complete `dict.pop()`, I get the docstring for `list.pop`)

Reply:
Yeah, some explanation might be in order here. I am keeping a flattened `Members` list of all methods and fields across all builtin types so that when we're not sure what type an expression is we can offer to complete any of the available methods. Because of some collisions this sort of thing happens.

Reply:
for cases where we know the type, we could still read off of Types, right?

Reply:
Yes, I think that's feasible. Looking at that now.